docs/getting-started/devenv/nodejs-development-tools.md

```diff
@@ -0,0 +1,50 @@
+# Node.js development tools
+
+When developing for the Block Editor, you will need [Node.js](https://nodejs.org/en) development tools along with a code editor and a local WordPress environment. Node.js (`node`) is an open-source runtime environment that allows you to execute JavaScript outside of the web browser.
+
+Installing `node` will automatically include Node Package Manager (`npm`) which includes the script `npx`, two tools you will frequently use in block and plugin development.
+
+Node Package Manager (`npm`) serves multiple purposes, including dependency management and script execution. It's the recommended package manager and is extensively featured in all documentation.
+
+The `npx` script is used to run commands from packages without installing them globally and is commonly used when scaffolding blocks with the [`create-block`](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-create-block/) package.
+
+
+## Mac and Linux installations
+
+For Mac and Linux, it's recommended that you use [Node Version Manager](https://github.com/nvm-sh/nvm) (`nvm`). This allows you to install and manage specific versions of `node`, which are installed locally in your home directory, avoiding any global permission issues.
+
+Here are the quick instructions for installing `node` using `nvm`. See the [complete installation guide](https://github.com/nvm-sh/nvm#installing-and-updating) for more details.
+
+1. Open the terminal and run the following to install `nvm`. On macOS, the required developer tools are not installed by default. Install them if prompted.
+
+```sh
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.5/install.sh | bash
+```
+
+2. Quit and restart the terminal.
+3. Run `nvm install lts` in the terminal to install the latest [LTS](https://nodejs.dev/en/about/releases/) (Long Term Support) version of Node.js.
```

Comment:
The actual command is `nvm install --lts` as per [this](https://github.com/nvm-sh/nvm#long-term-support)